portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Create/CreateGlobalPolicy.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import React from 'react';
+import Policies from 'AppComponents/GlobalPolicies/Policies/GlobalSpecificComponents/Policies';
+
+/**
+ * Global Policies Creating Page.
+ * @returns {JSX} - Creating Page.
+ */
+const CreateGlobalPolicy: React.FC = () => {
+    return (
+        <Policies isCreateNew policyID={null} disabled={false}/>
+    );
+};
+
+export default CreateGlobalPolicy;
```

Comment:
Let's add a new line to the bottom of the newly added files.

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)